### DISCUSSION TITLE
[VOTE] IPED-4.0.0-beta release candidate 1

### DISCUSSION THREAD

**Discussion by lfcnassif** [sepinf-inc/IPED#1178] [Polls]
All developers and contributors,

A release candidate for IPED-4.0.0-beta is available at:
https://github.com/sepinf-inc/IPED/actions/runs/2544090767

You can also pull and compile 4.0.0-beta-rc1 tag from:
https://github.com/sepinf-inc/IPED/tree/4.0.0-beta-rc1

Please vote on releasing this as IPED-4.0.0-beta.

[ ] +1 Release this package as IPED-4.0.0-beta
[ ] -1 Do not release this package as IPED-4.0.0-beta because...

Here is my +1.

Best,
Luís Filipe Nassif

**Comment by lfcnassif** [comment]
Please, I kindly ask for those that have already voted to vote again writing down your vote explicitly as an answer, so I would know if the vote comes from a developer or from the community, because voting user name isn't saved in the poll, I apologize for this (first time using github polls).

**Reply by wladimirleite** [reply]
I voted to release as beta.

**Reply by lfcnassif** [reply]
Given 2 +1 and no -1, I'll roll 4.0.0-beta. I'll also include the fixes for #1187 and #1186.

Thank you @tc-wleite.
